utils/fw_tunnel_updater.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import getpass
+import sys
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from rffe_nuttx_lib import RFFEFWUpdate
+from rffe_tunnel_lib import SSHTunnel
+
+
+def load_rack_config(file_path):
+    with open(file_path) as f:
+        return json.load(f)
+
+def get_getway(rack_id):
+    if rack_id == "TL":
+        return "ia-20rabpmtl-co-iocsrv.lnls-sirius.com.br"
+    else:
+        return f"ia-{rack_id}rabpm-co-iocsrv.lnls-sirius.com.br"
+
+def tunnel_forward_fw_update(gateway, ip, password, fw_file, version, key_path):
+    tunnel = SSHTunnel(
+            host=gateway,
+            username="lnls-bpm", 
```

Comment:
Although all CPUs in our MicroTCA crates have the `lnls-bpm` user, I think it would be better if it was an optional argument (with `lnls-bpm` as default).